rick.py

```diff
@@ -399,7 +399,7 @@ def line_chart(data, ylab, xlab, **kwargs):
 
         if (xticker_labels is not None):
             list_major_labels = xticker_labels
-            list_major_ticks  = xticker_slots
+            list_major_ticks  = np.arange(0, len(list_major_labels), 1)

```

Comment:
found in vfh_analysis/projects/2022_vfh_emissions/analysis/report_support/rick.py (branch: i17_1923_data)
added by scann0n

Reply:
I'm guessing @scann0n made the modification for this chart, which is the only one that uses the `xticker_labels` parameter to the `line_chart` function 
![Untitled](https://github.com/CityofToronto/bdit_plotting_gallery/assets/10623872/1e217711-69dd-4f80-9c99-a68080d60b21)

Note the increased number of months in the label compared to the version in the report

Reply:
Per our discussion yesterday, given that `xticker_slots` is user supplied, it should check if that variable is not None, and default to `np.arange(0, len(list_major_labels), 1)` if it is `None`